sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java

```diff
@@ -104,8 +127,10 @@ void recognizeReceiptDataWithContentTypeAutoDetection() {
      * Verifies receipt data for a document using source as as input stream data and text content when
      * includeTextDetails is true.
      */
-    @Test
-    void recognizeReceiptDataTextDetails() {
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+     public void recognizeReceiptDataTextDetails(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
```

Comment:
This test seems to be failing in live mode.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=386425&view=ms.vss-test-web.build-test-results-tab&runId=10847082&resultId=100047&paneView=debug